t04_refactored.py

```diff
@@ -140,9 +140,36 @@ def team_6_adv():
 
 
 def team_7_adv():
+    """
+    https://docs.google.com/document/d/1ksWilu6oBRWPoa5kZo2jyzkAOiAKqwtHYfMWGK7d27M/edit
+    Hira Karim
+    Claudia Pulido
+    :return:
+    """
     pass
```

Comment:
Remove this line, it is a placeholder for a function with no code in it. It can be removed now.